polaris.shopify.com/content/patterns/date-picking/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Date picking
+description: Makes it easy for merchants to select and input dates and date ranges.
+url: /patterns/date-picking
+previewImg: /images/patterns/pattern-thumbnail-date-picking.png
+status: {value: 'Beta', message: ''}
+order: 10
+githubDiscussionsLink: https://github.com/Shopify/polaris/discussions/7852
+variants:
+  - 'variants/single-date.md'
+  - 'variants/date-range.md'
+  - 'variants/date-list.md'
```

Comment:
This is 1000x better then everything in one file IMO. Lots of ways to do this though.